src/App.tsx

```diff
@@ -20,17 +27,90 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    itemWidth: 130,
+    frameSize: 3,
+    step: 3,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  handleNumber(key: NumKey) {
+    return (e: React.ChangeEvent<HTMLInputElement>) => {
+      const value = Number(e.target.value);
+      const parsed = Number.isNaN(value) ? 0 : value;
```

Comment:
if you want a shorter form of this
```suggestion
      const value = Number(e.target.value) || 0;
```
but your code looks more descriptive